cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -1239,6 +1245,10 @@ impl<I: VCodeInst> RegallocFunction for VCode<I> {
 
     fn is_ret(&self, insn: InsnIndex) -> bool {
         match self.insts[insn.index()].is_term() {
+            // TODO: this is probably not the place for this, we likely need an `is_trap` in the
```

Comment:
This seems fine IMHO; from the point-of-view of regalloc semantics, a trap might as well be a return (control leaves the function and there are no successors).

Reply:
Thanks, I'll remove this TODO :+1: